JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/scikitwrapper/ScikitLearnFeatureEngineeringWrapper.java

```diff
@@ -0,0 +1,67 @@
+package ai.libs.jaicore.ml.scikitwrapper;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.api4.java.ai.ml.core.dataset.supervised.ILabeledDataset;
+import org.api4.java.ai.ml.core.dataset.supervised.ILabeledInstance;
+import org.api4.java.ai.ml.core.evaluation.IPrediction;
+import org.api4.java.ai.ml.core.evaluation.IPredictionBatch;
+import org.api4.java.ai.ml.core.exception.TrainingException;
+
+import ai.libs.jaicore.basic.FileUtil;
+import ai.libs.jaicore.ml.core.EScikitLearnProblemType;
+
+public class ScikitLearnFeatureEngineeringWrapper<P extends IPrediction, B extends IPredictionBatch> extends AScikitLearnWrapper<P, B> {
```

Comment:
FeatureEngineering appears to me too general. This is already fixed to time series data to tabular isnt it?